service/src/test/java/bio/terra/workspace/service/workspace/flight/azure/DeleteAzureContextFlightTest.java

```diff
@@ -100,25 +100,25 @@ private void createAzureContext(UUID workspaceUuid, AuthenticatedUserRequest use
     azureCloudContextService.getRequiredAzureCloudContext(workspaceUuid);
   }
 
-  private UUID createAzureIpResource(UUID workspaceUuid, AuthenticatedUserRequest userRequest)
+  private UUID createAzureResource(UUID workspaceUuid, AuthenticatedUserRequest userRequest)
```

Comment:
So this test was just creating an arbitrary resource?

Reply:
correct, it just tests that things get deleted within a workspace